ProductApp/src/main/java/com/stack/jspdemo/controller/ProductController.java

```diff
@@ -0,0 +1,39 @@
+package com.stack.jspdemo.controller;
+
+import java.net.URI;
+import java.util.List;
+
+import javax.validation.Valid;
+
+import com.stack.jspdemo.model.Product;
+import com.stack.jspdemo.service.ProductService;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+
+@RestController
+public class ProductController {
+
+	@Autowired
+	ProductService service;
+
+	@GetMapping("/products")
+	public List<Product> retrieveAllProducts() {
+		return service.findAll();
+	}
+
+	@PostMapping("/products")
```

Comment:
Shall we specify what kind of message this endpoint consumes?

Reply:
Yes, Antonio explain me yesterday this part is incorrect and here I will set up a listener which will receive rest, process it and persist/list the payload, It will be ready when upload the rabbitmq part

Reply:
done